hack/common.sh

```diff
@@ -55,6 +55,8 @@ readonly OS_CROSS_COMPILE_PLATFORMS=(
 readonly OS_CROSS_COMPILE_TARGETS=(
   cmd/openshift
   cmd/oc
+  pkg/sdn/plugin/sdn-cni-plugin
```

Comment:
They would get built into the release.  You can always target a build directly with `hack/build-go.sh <path to package>`.  make / build-go shouldn't automatically build these i don't think